aten/src/ATen/native/LossNLL.cpp

```diff
@@ -183,7 +183,6 @@ static void nll_loss_out_frame(
   const target_t* target_data = target_contiguous.data_ptr<target_t>();
 
   const int64_t ndim = input.dim();
-  TORCH_CHECK(ndim <= 2);
```

Comment:
Why was this check removed?

Reply:
As these are structured kernels, all the checks happen there. Pretty sure they forgot to delete this when porting these functions to structured kernels.